app/templates/index.hbs

```diff
@@ -1,8 +1,10 @@
 <section class="container about">
   <div class="row">
     <div class="content-box col-md-7">
-      <h2>Managing Async With RxJS 5: Course Overview</h2>
-      <p>This is a 3 part course complete with pre-course videos, an interactive workshop day, and post workshop support.</p>
+      <h2>Managing Async With RxJS 5.5+: Course Overview</h2>
+      <p>This is a 2 part course complete with pre-course videos and an interactive pre-recorded 8 hour workshop.</p>
```

Comment:
@sivakumar-kailasam this says pre-recorded but does it describe all of them? should it say interactive workshop day?

Reply:
I took this from the eventbrite description. If you give a sentence in mind, I can update it.